app/Http/Requests/StoreVolumeMetadata.php

```diff
@@ -40,12 +40,12 @@ public function rules()
     {
         return [
             'metadata_csv' => [
-                'required_without_all:metadata_text,ifdo_file',
+                'bail:required_without_all:metadata_text,ifdo_file',
```

Comment:
`bail` must be added as a separate array element of the validation rules.

```suggestion
                'bail',
                'required_without_all:metadata_text,ifdo_file',
```

Reply:
ah okay. Can you maybe explain what the difference is between these two? It seemed to work for me.